main.py

```diff
@@ -148,6 +148,10 @@ def create_circuit(self):
         except ValueError as exception:
             return jsonify("Bad request: {}".format(exception)), 400
 
+        # verify duplicated evc
+        if self.is_duplicated_evc(evc):
+            return jsonify("Not Acceptable: This evc already exists."), 406
```

Comment:
It looks like the 406 error code has a [very specific meaning](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/406):

> 406 Not Acceptable client error response code indicates that a response
> matching the list of acceptable values defined in Accept-Charset
> and Accept-Language cannot be served.

**409 Conflict** seems to be a [better alternative](https://stackoverflow.com/questions/3290182/rest-http-status-codes-for-failed-validation-or-invalid-duplicate).

Reply:
Agreed with @hdiogenes , please @macartur make use of 409 error.

Reply:
Fixed.